src/replaceAds.ts

```diff
@@ -151,10 +134,11 @@ const replaceAds = async () => {
               action.parent.insertBefore(adElementWithWrapper, action.parent.children[action.insertionIndex]);
               break;
           }
-          if (shouldUseSliseAd) {
-            sliseAdMutationObserver.observe(adElement, { childList: true });
-            registerSliseAd(slotId);
+          if (shouldUseTKeyAd) {
+            provider = 'Temple Wallet';
+            loadedAdIntersectionObserver.observe(adElement);
           } else {
+            provider = 'HypeLab';
```

Comment:
I think we should send multiple events to the analytics if we have ads from multiple providers. However, it would be better to ask Dmytro Rusnak about this edge case